github-release.sh

```diff
@@ -125,7 +125,7 @@ You can find the full list of ${RELEASE_VERSION} changes [here](https://hibernat
 
 For additional details, see:
 
-- the [release page](https://hibernate.org/orm/releases/${RELEASE_VERSION_FAMILY}/)
+- the [release page](https://hibernate.org/${PROJECT}/releases/${RELEASE_VERSION_FAMILY}/)
 - the [Migration Guide](${migrationGuideUrl})
 - the [Introduction Guide](${introGuideUrl})
```

Comment:
I think this, and some of the content below, is invalid for Hibernate Reactive. You might want to use different content for each project?

Reply:
ok, I will revert it